rhpam79-image-streams.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ImageStreamList
 apiVersion: v1
 metadata:
-  name: rhpam78-image-streams
+  name: rhpam79-image-streams
```

Comment:
Here in image streams are not updated tags for images to 7.9

Reply:
could update, but, adocs will be updated after all prs, as it is auto generated.